core/domain/acl_decorators.py

```diff
@@ -1118,3 +1119,37 @@ def test_can_change_topic_publication_status(self, **kwargs):
     test_can_change_topic_publication_status.__wrapped__ = True
 
     return test_can_change_topic_publication_status
+
+
+def get_decorator_for_accepting_suggestion(decorator):
+    """Function that takes a decorator as an argument and then applies some
+    common checks and then checks the permissions specified by the passed in
+    decorator.
+
+    Args:
+        decorator: function. The decorator to be used to verify permissions
+            for accepting/rejecting suggestions.
+
+    Returns:
+        function. The new decorator which includes all the permission checks for
```

Comment:
A little vague. Can you be more descriptive of the checks that are included?

Reply:
Done